src/bin/cargo/subcommand_metadata.rs

```diff
@@ -0,0 +1,63 @@
+use std::path::Path;
+
+pub(crate) fn description(path: &Path) -> Option<String> {
```

Comment:
nit: I lean towards this being in `cargo-s`ubcommand-metadata` behind a feature flag so the setting and reading code is in one place as they are coupled together (even if that coupling is using standard OS procedures)